src/Http/Response.php

```diff
@@ -557,6 +557,7 @@ protected function _setHeader(string $header, string $value): void
     /**
      * Clear header
      *
+     * @phpstan-param non-empty-string $header
      * @param string $header Header key.
      * @return void
```

Comment:
```suggestion
     * @param string $header Header key.
     * @return void
     * @phpstan-param non-empty-string $header
```
We seem to keep these on the bottom. Should we continue?

Reply:
I guess a future sniffer might want to fix that instead of humans :)

Reply:
I don't really mind, but keeping @param and @return together is probably a good requirement.